build/quantities.mjs

```diff
@@ -454,7 +454,9 @@ var UNITS = {
   "<dozen>" :  [["doz","dz","dozen"],12.0,"prefix_only", ["<each>"]],
   "<percent>": [["%","percent"], 0.01, "prefix_only", ["<1>"]],
   "<ppm>" :  [["ppm"],1e-6, "prefix_only", ["<1>"]],
-  "<ppt>" :  [["ppt"],1e-9, "prefix_only", ["<1>"]],
+  "<ppb>" :  [["ppb"],1e-9, "prefix_only", ["<1>"]],
```

Comment:
These (beyond million) only make sense in the places like the US where 1 billion == 1 thousand million, not in places like the UK where 1 billion == 1 million million.

Reply:
Please ignore the generated files. I did not write this code, but they weren't .gitignored so I felt like I should include them.